ui-tests/cypress/support/pageObjects/pages/templates/AddPage.js

```diff
@@ -4,15 +4,15 @@ import TemplatesPage from '../../pages/templates/TemplatesPage';
 import { DATA_TESTID, htmlElements } from '../../WebElement';
 
 export default class AddPage extends Content {
-  codeInput = `${htmlElements.input}[name="code"][${DATA_TESTID}=formik-field_RenderTextInput_input]`;
-  nameInput = `${htmlElements.input}[name="descr"][${DATA_TESTID}=formik-field_RenderTextInput_input]`;
-  jsonConfigDiv = `[${DATA_TESTID}=formik-field_JsonCodeEditorRenderer_div]`;
-  templateDiv = `[${DATA_TESTID}=formik-field_HtmlCodeEditorRenderer_div]`;
+  codeInput = `${htmlElements.input}[name="code"]#code`;
+  nameInput = `${htmlElements.input}[name="descr"]#descr`;
+  jsonConfigDiv = `${htmlElements.div}.form-group`;   //I don't like that these two are the same, but there's no other identifier
+  templateDiv = `${htmlElements.div}.form-group`;     //A solution could be to merge them into a "codeMirrorDiv"
```

Comment:
Merge them, it doesn't make sense that we have two identifier for the same element

Reply:
Merged them into a single "codeMirrorDiv"